test/test_dcrt.py

```diff
@@ -2,11 +2,12 @@
 Test the dcrt module
 """
 
-from hidimstat.dcrt import dcrt_zero
+import pytest
 import numpy as np
 from sklearn.covariance import LedoitWolf
 from sklearn.datasets import make_regression, make_classification
-import pytest
+
+from hidimstat.dcrt import dcrt_zero, dcrt_pvalue, _lasso_distillation_residual
 
 
 def test_dcrt_lasso():
```

Comment:
This test is way too long. I think we would gain  a lot in readability if we split it into small tests.

Note that test code is meant to be read, too !

Reply:
done